automated/linux/usb/test-usb-2.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+PATH_DEV="/dev/sda1"
+PATH_MOUNT="/mnt/lava_usb_test"
+
+lsblk | grep sda1
+if [ $? -eq 0 ]
+then
+    #Flash disk will be formatted
+    mkfs.ntfs "$PATH_DEV"
```

Comment:
Why ntfs and not ext3/4?
Can we assume, that the mkfs-tools, especially the ntfs-3g package, are installed? How about e2fsprogs? 
We should document every tool that is needed for the tests so that we can create a customized image for the workers.